src/components/text-editor/text-editor.tsx

```diff
@@ -111,6 +112,16 @@ export class TextEditor implements FormComponent<string> {
     @Prop({ reflect: true })
     public allowResize: boolean = true;
 
+    /**
+     * Defines how the editor should look
+     * - `default`: The default look of the editor
+     * - `compact`: A more compact look of the editor, useful for
+     * displaying on mobile devices for instance space is limited.
+     * It will remove the toolbar, unless the editor is in focus.
+     */
+    @Prop({ reflect: true })
+    public ui?: 'default' | 'compact' = 'default';
```

Comment:
Does this need to be a `@Prop`? It seems to only affect styling, can we not do the same thing with a class?

Reply:
We could for now. But I am not a big fan of that approach, unless we want to hide it from consumers.
Because:
1. if we add more functionality or complexities in future, having a prop makes it way easier to control these things. 
2. it's hard to communicated and test these things in the docs. Also it's harder for consumers to add classes. I'd prefer that they play around with props instead of concepts that are "styles-related".

If there is something that we only internally want to use, I think we can do that with the class.